vars/runIntegTests.groovy

```diff
@@ -0,0 +1,27 @@
+void call(Map args = [:]) {
+    git url: 'https://github.com/opensearch-project/opensearch-build.git', branch: 'main'
+
+    def lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+    String manifest = args.manifest ?: "manifests/${INPUT_MANIFEST}"
+    def inputManifest = lib.jenkins.InputManifest.new(readYaml(file: manifest))
+
+    String tarball = args.architecture == "x64" ? env.ARTIFACT_URL_linux_x64 : env.ARTIFACT_URL_linux_arm64
+    String artifactRootUrl = args.architecture == "x64" ? env.ARTIFACT_ROOT_URL_linux_x64 : env.ARTIFACT_ROOT_URL_linux_arm64
+    String buildManifest = "${artifactRootUrl}/dist/opensearch/manifest.yml"
```

Comment:
Again we could, but it would mean adding a param in `distribution-build.jenkinsfile`. What do you think about putting this logic into a method in `BuildManifest.groovy` called something like `getUrl()` which could build and return the full path? Then if there's future changes, it would only be a single change. `getArtifactRootUrl()` already exists so it would logically fit there.

Reply:
👍 Great suggestion, I like it

Reply:
Added as `getDistUrl()` in https://github.com/opensearch-project/opensearch-build/pull/1429/commits/15d3640d54eaddc6ec2b130cbf6575b50f2a7f32